app/views/doc/ext.html.erb

```diff
@@ -34,6 +34,12 @@
             ( <a href="https://backlog.com/git-tutorial/kr/">Korean</a> / <a href="https://backlog.com/git-tutorial/cn/">Simplified Chinese</a> / <a href="https://backlog.com/git-tutorial/tw/">Traditional Chinese</a> / <a href="https://backlog.com/git-tutorial/vn/">Vietnamese</a> )
           </p>
         </li>
+        <li>
+          <h4><a href="http://git.github.io/git-reference/">Git Reference</a></h4>
+          <p class='description'>
+            The Git Reference site is meant to be a quick reference for learning and remembering the most important and commonly used Git commands.
+          </p>
+        </li>
```

Comment:
I think we decided not to link to this unmaintained resource, per https://github.com/git/git-scm.com/pull/1083#discussion_r154743752.